app/presenters/publishing_api_presenters/html_attachment.rb

```diff
@@ -1,6 +1,11 @@
 require_relative "../publishing_api_presenters"
 
 class PublishingApiPresenters::HtmlAttachment < PublishingApiPresenters::Item
+  def initialize(item, update_type: nil)
+    super
+    item.govspeak_content.render_govspeak!
```

Comment:
I feel like perhaps this should only be called on during the `body` method, not when instantiating a class